shared/http/httpClientValues.ts

```diff
@@ -0,0 +1,34 @@
+enum REST_VERBS {
+  GET = "GET",
+  POST = "POST",
+  PUT = "PUT",
+  DELETE = "DELETE",
+  PATCH = "PATCH",
+}
+
+enum HTTP_STATUS_CODES {
+  OK = 200,
+  CREATED = 201,
+  ACCEPTED = 202,
+  NO_CONTENT = 204,
+  BAD_REQUEST = 400,
+  UNAUTHORIZED = 401,
+  FORBIDDEN = 403,
+  NOT_FOUND = 404,
+  METHOD_NOT_ALLOWED = 405,
+  CONFLICT = 409,
+  INTERNAL_SERVER_ERROR = 500,
+  NOT_IMPLEMENTED = 501,
+}
+
+enum HTTP_HEADERS {
+  CONTENT_TYPE = "Content-Type",
+  ACCEPT = "Accept",
+  AUTHORIZATION = "Authorization",
+}
+
+enum HTTP_CONTENT_TYPES {
+  APPLICATION_JSON = "application/json",
+}
+
+export { REST_VERBS, HTTP_STATUS_CODES, HTTP_HEADERS, HTTP_CONTENT_TYPES };
```

Comment:
needed to double check whether this is now redundant (and if not whether express etc. provides any/all of these constants)

Reply:
This is used in the Auth middleware. Express does not provide a similar set of const / enums. Google reveals some libraries which are doing that same thing as we are doing here.